### ISSUE TITLE
Model 1

### ISSUE THREAD

**Issue by jacaseyclyde** [sjsu-high-z-classify/GalNet#24]
**Model architecture.**

Layer | Type | Size | Features |  Kernel | Stride | Padding | Activation | Notes
--------|--------|-------|-------------|-----------|--------|------------|-------------|---------
In | Input | 69x69 | 3 | | | | | Input Image
1 | Conv2D | 64x64 | 32 | 6x6 | 1 | | Valid | ReLU |
2 | MaxPool2D | 32x32 | 32 | 2x2 | 2 | Valid | |
3 | Conv2D | 28x28 | 64 | 5x5 | 1 | Valid | | | ReLU |
4 | MaxPool2D | 14x14 | 64 | 2x2 | 2 | Valid | | |
5 | Conv2D | 11x11 | 128 | 3x3 | 1 | Valid | | ReLU |
6 | Conv2D | 8x8 | 128 | 3x3 | 1 | Valid | | ReLU |
7 | MaxPool2D | 4x4 | 128 | 2x2 | 2 | Valid | | |
8 | Dense | 2048 | | | | | | Flattened layer
9 | Dense | 64 | | | | | ReLU |
Out | Dense | 3 | | | | | Softmax | Output prediction

**Hyperparameters**
Optimizer: Adam
Loss: categorical_crossentropy
Loss Weights: None
Metrics: accuracy

**Describe alternative hyperparameters**
Optimizer: Nadam
Loss Weights: agreement

**Additional context**
Use dropout=.5 on all hidden Dense layers. Images are automatically scaled from original size (207x207) to input dimensions (69x69). Architecture based on combination of Dieleman and Dominguez-Sanchez.

**References**
<to be updated>

**Comment by jacaseyclyde** [comment]
Model architecture (note it's slightly different in size from what I outlined above - chalk it up to algebra errors):
![model](https://user-images.githubusercontent.com/7811666/60910163-e1b14100-a234-11e9-99b9-cb7f533d97f6.png)

**Comment by jacaseyclyde** [comment]
Batch Size: 256
Input Dimensions: 69x69
Epochs: 100
Optimizer: Adam
Learning Rate: 0.0001
Loss: categorical_crossentropy (keras)

Training Accuracy:
![acc](https://user-images.githubusercontent.com/7811666/60910205-f988c500-a234-11e9-986b-9a33bcaef414.png)

Training Loss:
![val](https://user-images.githubusercontent.com/7811666/60910219-04dbf080-a235-11e9-86de-0f5af58cd4bd.png)
Test Results:
    loss: 0.1383
    acc: 0.9446

**Comment by jacaseyclyde** [comment]
Batch Size: 256
Input Dimensions: 69x69
Epochs: 100
Optimizer: Nadam
Initial Learning Rate: 0.0001
Loss: categorical_crossentropy (keras)

Training Accuracy:
![acc](https://user-images.githubusercontent.com/7811666/61002303-1ea13480-a316-11e9-9a2c-a898a36da296.png)

Training Loss:
![loss](https://user-images.githubusercontent.com/7811666/61002318-282a9c80-a316-11e9-94a9-4926884de440.png)

Test Results:
loss: 0.1528
acc: 0.9410

**Comment by jacaseyclyde** [comment]
GPUs: 2
Batch Size: 256
Input Dimensions: 69x69
Epochs: 100
Optimizer: Nadam
Initial Learning Rate: 0.0001
Loss: categorical_crossentropy (keras)

Training Accuracy:
![acc](https://user-images.githubusercontent.com/7811666/61002400-54461d80-a316-11e9-9426-accd1acba81e.png)

Training Loss:
![loss](https://user-images.githubusercontent.com/7811666/61002419-5c9e5880-a316-11e9-9301-7536f02f6c1c.png)

Test Results:
loss: 0.1307
acc: 0.9475

**Comment by jacaseyclyde** [comment]
This particular model has probably reached its limit, though future models should train for at least 200 iterations
